integration-tests/README.md

```diff
@@ -0,0 +1,15 @@
+## Steps to Run Locally
+1. Install docker (if not already installed)
+2. Open a command prompt
+3. Clone this repository
+4. `cd <REPO_ROOT>/integration-tests`
+5. `docker compose up -d --project-directory docker-sws-api`
```

Comment:
Please update this to be consistent with the other APIs.
- Remove  the `-d`
- Add `--force-recreate`
- Include the note that "This is being run without the detached (-d) option so that it can be easier to stop. It may be run either way."

I realize this seems like hair-splitting but not everyone has the same knowledge of Docker and we've seen experienced people get tripped up by not realizing they need to do `docker compose down` to completely unload/reset the Elasticsearch container.